config_example.sh

```diff
@@ -118,7 +118,7 @@
 #
 # Set libvirt firmware and BMC bootMode
 # Choose "legacy" (bios), "UEFI", or "UEFISecureBoot"
-# Defaults to legacy for ipv4, UEFI for ipv6
+# Defaults to uefi
```

Comment:
```suggestion
# Defaults to UEFI
```
Since its case-sensitive, it would be nice to have it capitalized as it was before.

Reply:
done